### PR TITLE
Updated readme header & added badges

### PR DESCRIPTION
**Pull request by wandadars**
Please fill in the issue number this pull request is fixing:

Fixes #

Changes proposed in this pull request:
- Updated the readme file to use the Cantera logo and added some badges
-
-

### CODE REVIEW

**Comment by bryanwweber** [comment]
FYI for the future - you don't have to close a PR and make a new one, you can just force push to the existing branch and the PR will be updated

**Comment by wandadars** [comment]
I was making the edits in the Github web interface, and I got really tangled up when I accidentally caused a merge conflict. I had like 5 different branches at one point, and I accidentally made the pull request on the wrong one(while thinking it was the correct one). Then I deleted that branch and  Github auto-closed the PR. lol.

File: README.rst [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+   :width: 675px
+   :align: center
+   
 
-.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
+.. |doi| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
    :target: https://doi.org/10.5281/zenodo.1174508
+   
+.. |codecov| image:: https://img.shields.io/codecov/c/github/Cantera/cantera/master.svg
+   :target: https://codecov.io/gh/Cantera/cantera?branch=master
```
Comment (speth) [inline README.rst]: I'm finding these `shields.io` badges take an exceedingly long time to load and frequently time out. They're cute, but do they really add anything?

File: README.rst [hunk 2]
```diff
@@ -126,15 +149,13 @@ Summary:
...
-.. image:: https://travis-ci.org/Cantera/cantera.svg?branch=master
-    :target: https://travis-ci.org/Cantera/cantera
-
-Appveyor builds (Windows):
+==============  ============  ===================
+Platform        Site          Status
+==============  ============  ===================
+Linux & OS X    Travis CI     |travisci|
+Windows x64     Appveyor      |appveyor|
+==============  ============  ===================
```
Comment (speth) [inline README.rst:132]: I like this -- I hadn't realized that using the `|label|` substitutions would let you put images into a table.

File: README.rst [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/auhd35qn9cdmkpoj?svg=true
+    :target: https://ci.appveyor.com/project/Cantera/cantera
+
+.. |release| image:: https://img.shields.io/github/release/cantera/cantera.svg   
+   :alt: GitHub release
+
+.. |commits| image:: https://img.shields.io/github/commits-since/Cantera/cantera/latest.svg   
+   :alt: Github commits (since latest release)
+
+|doi| |codecov| |travisci| |appveyor| |release| |commits|
```
Comment (speth) [inline README.rst]: My other concern with this is that it makes this file much less readable anywhere except on the Github landing page. I like the fact that the current version has a minimal amount of `rst` cruft at the top before you get to human-readable content.

**Review by speth — CHANGES_REQUESTED** [review]
I'd like to better understand what the goal of these revisions is before proceeding.

File: README.rst (hunk 1 above)
Comment (wandadars) [reply README.rst]: Some of them are not absolutely critical. I personally find them useful on other repos for making quick overviews of the state of a github project.

Comment (wandadars) [reply README.rst]: The Github commits badge can be removed as it does not convey a very rich project metric.

**Comment by wandadars** [comment]
@speth I have changed the location of the shields.io badge definitions to be at the bottom of the readme to help with the clarity of the file in raw viewing format.

File: README.rst [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
-Cantera
-*******
+.. Cantera
 
-Version 2.4.0b1 (development)
+.. image:: http://cantera.github.io/docs/sphinx/html/_static/cantera-logo.png
+   :target: http://cantera.github.io/docs/sphinx/html/index.html#
+   :alt: cantera logo
+   :width: 675px
+   :align: center
```
Comment (speth) [inline README.rst]: Can you move this to the bottom and just leave a `|Cantera|` label here?

File: README.rst [hunk 5]
```diff
@@ -145,3 +147,20 @@ supporting the open source scientific computing community.
...
+    :target: https://ci.appveyor.com/project/Cantera/cantera
+
+.. |doi| image:: https://zenodo.org/badge/DOI/10.5281/zenodo.170284.svg
+   :target: https://doi.org/10.5281/zenodo.1174508
+    
+.. |codecov| image:: https://img.shields.io/codecov/c/github/Cantera/cantera/master.svg
+   :target: https://codecov.io/gh/Cantera/cantera?branch=master
+
+.. |release| image:: https://img.shields.io/github/release/cantera/cantera.svg   
+   :alt: GitHub release
```
Comment (speth) [inline README.rst]: So, this currently just links to the image? Shouldn't it link to https://github.com/Cantera/cantera/releases?

File: README.rst [hunk 6]
```diff
@@ -93,6 +95,8 @@ and we will address it as soon as possible.
 Development Site
 ================
 
+:Version: 2.4.0b1 (development)
```
Comment (speth) [inline README.rst]: At this point, this information may as well just be part of the prose of this paragraph.

**Review by speth — CHANGES_REQUESTED** [review]

**Comment by wandadars** [comment]
@speth I moved the image definition to the bottom and also changed the link for the release badge. Also moved that version note into the paragraph.

**Review by speth — APPROVED** [review]

**Pull request merged**